index.js

```diff
@@ -24,12 +24,31 @@ try {
   const logoPosition = core.getInput('logoPosition');
   const style        = core.getInput('style');
   const cacheSeconds = core.getInput('cacheSeconds');
+  const valColorRange = core.getInput('valColorRange');
+  const minColorRange = core.getInput('minColorRange');
+  const maxColorRange = core.getInput('maxColorRange');
+  const invertColorRange = core.getInput('invertColorRange');
 
   if (labelColor != '') {
     content.labelColor = labelColor;
   }
 
-  if (color != '') {
+  if (minColorRange != '' && maxColorRange != '' && valColorRange != '') {
+    var max = parseFloat(maxColorRange);
+    var min = parseFloat(minColorRange);
+    var val = parseFloat(valColorRange);
+    if (val < min) val = min;
+    if (val > max) val = max;
+    var hue = 0
+    if (invertColorRange == '') {
+      hue = Math.floor((val - min) / (max - min) * 100);
+      content.message = val + " / " + max + " " + content.message
+    } else {
+      hue = Math.floor((max - val) / (max - min) * 100);
+      content.message = valColorRange + " " + content.message
+    }
```

Comment:
I guess setting the `content.message` was just for debugging purposes? Also, I think green has a hue of 120, so I would suggest to replace the `100` with a `120`.

```suggestion
    if (invertColorRange == '') {
      hue = Math.floor((val - min) / (max - min) * 120);
    } else {
      hue = Math.floor((max - val) / (max - min) * 120);
    }
```

Reply:
I agree with the change in the green value. 
About the `content.message`, it was not for debugg, but to update the badge message based on the values given. Here is a snapshot from the repository I was using to test the workflow (https://github.com/LucasWolfgang/QAS-Editor)
![image](https://user-images.githubusercontent.com/39681420/163676525-f4ec19f5-0349-46f4-9985-7f81f4ef2efe.png)

After thinkig a bit, I concluded that, even so, this change is ok, since it is possible to manually do this while pasing the parameters.

Reply:
Knowing this, do you prefer to keep the changed or roll it back?

Reply:
I think it would be more flexible if the message is not set automatically. Else, we would also need to make the `message` parameter non-mandatory and this would become quite confusing...

Reply:
I agree